multiversx_sdk/validators/validators_file.py

```diff
@@ -0,0 +1,62 @@
+import json
+from pathlib import Path
+
+from multiversx_sdk.validators.errors import (
+    CannotReadValidatorsDataError,
+    ValidatorsFileNotFoundError,
+)
+from multiversx_sdk.wallet.validator_keys import ValidatorPublicKey
+from multiversx_sdk.wallet.validator_pem import ValidatorPEM
+from multiversx_sdk.wallet.validator_signer import ValidatorSigner
+
+
+class ValidatorsFile:
```

Comment:
We should also define this in specs.

Refactoring idea:

```
ValidatorsFile or BunchOfValidators:
    class method: new_from_file() - this one completely loads PEM files / signers and holds them as private members (thus, eager load, instead of lazy load).
    get_public_keys(), instead of load
    get_signers(), instead of load
    etc.
```

Reply:
How about also allowing the following named constructor?

```
ValidatorsFile.new_from_pem()
```

Since validator PEM files can also hold more secret keys.

Reply:
Can also be named `ValidatorsKeysHolder`, and be held in wallet package? :thought_balloon: